### ISSUE TITLE
Ticking block entity

### ISSUE THREAD

**Issue by MrMinecart444** [TeamAOF/All-of-Fabric-6#230]
Good day
we have the problem on our server that the server crashes when we want to join the server or the server no longer starts. The crash log states that it is a ticking block entity error which leads back to a glassfiber_cable.
Can it be that there is somehow an error in the mod?

This is the bug report
-- Block entity being ticked --
Details:
Name: techreborn:cable // techreborn.blockentity.cable.CableBlockEntity
Block: Block{techreborn:glassfiber_cable}[covered=false,down=false,east=false,north=true,south=true,up=true,waterlogged=false,west=false]
Block location: World: (1296,69,-186), Section: (at 0,5,6 in 81,4,-12; chunk contains blocks 1296,-64,-192 to 1311,319,-177), Region: (2,-1; contains chunks 64,-32 to 95,-1, blocks 1024,-64,-512 to 1535,319,-1)
Block: Block{techreborn:glassfiber_cable}[covered=false,down=false,east=false,north=true,south=true,up=true,waterlogged=false,west=false]
Block location: World: (1296,69,-186), Section: (at 0,5,6 in 81,4,-12; chunk contains blocks 1296,-64,-192 to 1311,319,-177), Region: (2,-1; contains chunks 64,-32 to 95,-1, blocks 1024,-64,-512 to 1535,319,-1)
Stack trace:
at net.minecraft.class_2818$class_5563.method_31703(class_2818.java:660)
at net.minecraft.class_2818$class_5564.method_31703(class_2818.java:714)
at net.minecraft.class_1937.method_18471(class_1937.java:456)
at net.minecraft.class_3218.method_18765(class_3218.java:376)
at net.minecraft.server.MinecraftServer.method_3813(MinecraftServer.java:875)
at net.minecraft.server.MinecraftServer.method_3748(MinecraftServer.java:819)
at net.minecraft.class_1132.method_3748(class_1132.java:98)
at net.minecraft.server.MinecraftServer.method_29741(MinecraftServer.java:665)
at net.minecraft.server.MinecraftServer.method_29739(MinecraftServer.java:257)
at java.base/java.lang.Thread.run(Thread.java:833)

-- Affected level --
Details:
All players: 1 total; [class_3222['MrMinecart444'/199, l='ServerLevel[Server2]', x=1965.53, y=64.00, z=-648.00]]
Chunk stats: 4686
Level dimension: minecraft:overworld
Level spawn location: World: (1281,89,-64), Section: (at 1,9,0 in 80,5,-4; chunk contains blocks 1280,-64,-64 to 1295,319,-49) , Region: (2,-1; contains chunks 64,-32 to 95,-1, blocks 1024,-64,-512 to 1535,319,-1)
Level time: 11281030 game time, 15023614 day time
Level name: Server2
Level game mode: Game mode: survival (ID 0). Hardcore: false. Cheats: false
Level weather: Rain time: 30840 (now: false), thunder time: 5182 (now: false)
Known server brands: fabric
Level was modded: true
Level storage version: 0x04ABD - Anvil
Stack trace:
at net.minecraft.server.MinecraftServer.method_3813(MinecraftServer.java:875)
at net.minecraft.server.MinecraftServer.method_3748(MinecraftServer.java:819)
at net.minecraft.class_1132.method_3748(class_1132.java:98)
at net.minecraft.server.MinecraftServer.method_29741(MinecraftServer.java:665)
at net.minecraft.server.MinecraftServer.method_29739(MinecraftServer.java:257)
at java.base/java.lang.Thread.run(Thread.java:833)

**Comment by James103** [comment]
Do you have the full crash report (with the exception)? If so, can you please post it here?

**Comment by LiNoKami** [comment]
This issue appeared for us, when we connected glassfiber_cable with powah cables and have machines on the powah side.

Here is a scuffed solution we used to solve this problem:
We figured out the coordinates in which the cables are located and then replaced all cables with for example stone in the area via the console. The server was capable of executing commands between loading the world and finally crashing. Hope it works for you :)

The commands we used:
/fill 920 40 -800 960 60 -770 stone replace techreborn:glassfiber_cable
/fill 920 40 -800 960 60 -770 stone replace powah:energy_cable_nitro
